addons/account/views/account_report.xml

```diff
@@ -9,7 +9,7 @@
             report_type="qweb-pdf"
             name="account.report_invoice_with_payments"
             file="account.report_invoice_with_payments"
-            attachment="(object.state in ('open','in_payment','paid')) and ('INV'+(object.number or '').replace('/','')+'.pdf')"
```

Comment:
You are doing the opposite in https://github.com/odoo/odoo/pull/46401

Reply:
indeed, I found 939bf9a25dd27608fd17217b8ba83931e2c37eff afterward, while working on the other pr

Reply:
by the way, not needed anymore if we are using the checksum.

Reply:
I'd remove that part since we don't rely on the name anymore for the fix